README.md

```diff
@@ -16,38 +15,82 @@
 </p>
 
 **Qdrant** (read: _quadrant_ ) is a vector similarity search engine and vector database.
-It provides a production-ready service with a convenient API to store, search, and manage points - vectors with an additional payload.
-Qdrant is tailored to extended filtering support.  It makes it useful for all sorts of neural-network or semantic-based matching, faceted search, and other applications. 
+It provides a production-ready service with a convenient API to store, search, and manage points—vectors with an additional payload
+Qdrant is tailored to extended filtering support. It makes it useful for all sorts of neural-network or semantic-based matching, faceted search, and other applications.
 
-Qdrant is written in Rust 🦀, which makes it fast and reliable even under high load.
+Qdrant is written in Rust 🦀, which makes it fast and reliable even under high load. See [benchmarks](https://qdrant.tech/benchmarks/).
 
 With Qdrant, embeddings or neural network encoders can be turned into full-fledged applications for matching, searching, recommending, and much more!
 
-Also available as managed solution in the **Qdrant Cloud** https://cloud.qdrant.io/ ⛅
+<p align="center">
+<strong><a href="../QUICK_START.md">Qdrant Quick Start with Docker</a> • <a href="https://qdrant.tech/documentation/cloud/cloud-quick-start/">Use Managed Qdrant Cloud</a> • <a href="#clients">Programming Language Clients</a> • <a href="#demo-projects">Demo Projects</a> • <a href="#integrations">Integrations</a> • <a href="#Contacts">Contact</a>
 
-## Demo Projects
+</strong>
+</p>
 
-### Semantic Text Search 🔍
+## Getting Started
+
+## Python
+
+```
+pip install qdrant-client
+```
+
+The client offers a convenient way to interact with Qdrant from Python across 3 different modes:
+
+```python
+from qdrant_client import QdrantClient
+qdrant = QdrantClient(":memory:") # Create in-memory Qdrant instance, for testing, CI/CD
+# OR
+client = QdrantClient(path="path/to/db")  # Persists changes to disk, fast prototyping
+```
+
+## Client Server
```

Comment:
Not super clear that this is the 3rd mode, since it has its own title

Reply:
https://github.com/qdrant/qdrant/pull/1793#discussion_r1179112015

Reply:
Addressed and changed